.expeditor/end_to_end.pipeline.yml

```diff
@@ -256,6 +256,17 @@ steps:
             - BUILD_PKG_TARGET=x86_64-linux
             - HAB_BLDR_URL=https://bldr.acceptance.habitat.sh
 
+  - label: "[:windows: test_windows_service_stops_on_launcher_termination]"
+    command:
+      - powershell .expeditor/scripts/end_to_end/run_e2e_test.ps1 DEV test_windows_service_stops_on_launcher_termination
```

Comment:
You'll want to use "dev" instead of "DEV" from here on out; see https://github.com/habitat-sh/habitat/pull/7100 for more.

Reply:
oh ok. I am also making the replacement in a few files that got missed.